node/src/gql/mutation/me/update-answer.ts

```diff
@@ -53,7 +56,10 @@ export const updateAnswer = {
     if (!mentor) {
       throw new Error(`no mentor found for id '${args.mentorId}'`);
     }
-    if (`${context.user._id}` !== `${mentor.user}`) {
+    if (
+      `${context.user._id}` !== `${mentor.user}` &&
+      context.user._id !== requireEnv('API_SECRET')
```

Comment:
can we get this to work without setting the API_SECRET as `context.user._id`? That seems like it's asking for trouble